tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -97,6 +97,8 @@ EventsAndTriggers:
           - PointwiseL2Norm(ThreeIndexConstraint)
           - PointwiseL2Norm(FourIndexConstraint)
         InterpolateToMesh: None
+        CoordinatesFloatingPointType: Float
```

Comment:
For my understanding: why are you changing the behavior in this input file (and in the Xcts/Schwarzschild file)?

Reply:
probably some copy-paste thing or something. Changing to `double`